### ISSUE TITLE
Automatic PRs should use a proxy when referring to PRs

### ISSUE THREAD

**Issue by oliviertassinari** [snyk/user-docs#370]
I'm reopening a new issue for #357 as it wasn't fixed.

**Comment by TashaTBaker** [comment]
Hi @oliviertassinari ,
I think this is something our support team or your account manager can help you with. Please [log a ticket](https://support.snyk.io/hc/en-us/requests/new) and link them to the issue you raised previously on this repo and they will be able to assist you.
Thank you!

**Comment by oliviertassinari** [comment]
Ok, issue open and tweet sent, hopefully, it will be enough. Thanks https://x.com/olivtassinari/status/1843715459402871151

**Comment by Jdunsby** [comment]
Hi @oliviertassinari

Massive thanks for persisting with this issue and I'm very sorry for the lack of response on your previous issue 🙏 

It seems our last solution had some holes in it, but we've released a fix which should take care of the remaining problems. The fix uses the official Github redirect link (`redirect.github.com`) to avoid the case you mentioned in your previous issue where a custom redirect link could be blocked by Github. The limitation of this fix is that it will only apply to new PRs opened by Snyk.

Thanks again for your patience on this issue, and please do let us know if the problem reoccurs, or if you have similar problems in future. I have replied on your support ticket too for visibility.

**Comment by oliviertassinari** [comment]
@Jdunsby Great, thanks taking this on.

**Comment by Jdunsby** [comment]
Glad to hear it's been resolved 🎉
Thanks again for flagging this.
